files/find.py

```diff
@@ -313,7 +313,7 @@ def main():
                        continue
 
                     try:
-                        st = os.stat(fsname)
+                        st = os.lstat(fsname)
```

Comment:
shouldn't this be conditioned on 'follow'? as this now breaks the opposite case (follow=yes)